app/components/static/About.js

```diff
@@ -90,6 +95,24 @@ export default class About extends Component<any> {
 
     const apiIcon = environment.isAdaApi() ? cardanoIcon : mantisIcon;
 
+    let platform;
+    switch (environment.platform) {
+      case 'darwin':
+        platform = 'macOS';
+        break;
+      case 'win32':
+        platform = 'Windows';
```

Comment:
Are we sure that the names of the OSes are universally used in all languages (without translation)?

Reply:
No we are not :)
I have requested the translations and if the Japanese translation is different then I will turn these hardcoded values into translation-keys.
The same is for the Cardano/Mantis names...

Reply:
@DominikGuzei I've received translations and it turns out macOS/Windows/Linux is the same in Japanese as in English so I will leave it like is.